flows/actions/testdata/send_msg.json

```diff
@@ -917,5 +912,102 @@
             "waiting_exits": [],
             "parent_refs": []
         }
+    },
+    {
+        "description": "Template with header image component",
+        "action": {
+            "type": "send_msg",
+            "uuid": "ad154980-7bf7-4ab8-8728-545fd6378912",
+            "text": "The Maine Coone is the only native American long haired breed.",
+            "attachments": [
+                "image/jpeg:http://example.com/cat1.jpg"
+            ],
+            "template": {
+                "uuid": "be68beff-1a5b-424b-815e-023cc53c1ddc",
+                "name": "cat_fact"
+            },
+            "template_variables": [
+                "http://example.com/cat2.jpg",
```

Comment:
@norkans7 @ericnewcomer wondering if this shouldn't be `"image/jpeg:http://example.com/cat2.jpg"` so that these image variables are more consistent with how we do msg media in general.. and then the onus is on courier to transform it like it would a regular attachment.

Reply:
I adjusted courier side for that in https://github.com/nyaruka/courier/pull/752

Reply:
Ok I've updated this PR to assume that variable values look like attachments